components/helper/ShareButtons.tsx

```diff
@@ -9,9 +11,133 @@ import {
   TwitterShareButton,
 } from 'react-share';
 
-export default function ShareButtons() {
+type ShareButtonsProps = {
+  popup: boolean;
+  setCopied: React.Dispatch<React.SetStateAction<boolean>>;
+  copied: boolean;
+};
+
+export default function ShareButtons({
+  popup, setCopied, copied,
+}: ShareButtonsProps) {
   const url = 'https://clearviction.org/calculator/head-initial-1-cont';
 
+  const style = {
+    fontSize: '1.25rem',
```

Comment:
`body1` in the theme file already has several of these as part of its already existing style, so we can clean this up a little further.

Reply:
Makes sense. I have left only what is completely necessary